openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -26411,3 +26411,367 @@ OS:Meter:CustomDecrement,
+  A4, \field Update Frequency
+       \required-field
+       \type choice
+       \key ZoneTimestep
+       \key SystemTimestep
+  A5, \field EMS Program or Subroutine Name
+       \type object-list
+       \object-list ErlProgramSubroutineNames
+       \note optional for global scope variables, required for local scope variables
+  A6, \field Resource Type
+       \required-field
+       \note choose the type of fuel, water, electricity, pollution or heat rate that should be metered.
+       \type choice
+       \key Electricity
+       \key NaturalGas
+       \key Gasoline
+       \key Diesel
+       \key Coal
+       \key FuelOil#1
+       \key FuelOil#2
+       \key Propane
+       \key OtherFuel1
+       \key OtherFuel2
+       \key WaterUse
+       \key OnSiteWaterProduced
+       \key MainsWaterSupply
+       \key RainWaterCollected
+       \key WellWaterDrawn
+       \key CondensateWaterCollected
+       \key EnergyTransfer
+       \key Steam
+       \key DistrictCooling
+       \key DistrictHeating
+       \key ElectricityProducedOnSite
+       \key SolarWaterHeating
+       \key SolarAirHeating
+  A7, \field Group Type
+       \note choose a general classification, building (internal services), HVAC (air systems), or plant (hydronic systems), or system
+       \type choice
+       \required-field
+       \key Building
+       \key HVAC
+       \key Plant
+       \key System
+  A8, \field End-Use Category
+       \note choose how the metered output should be classified for end-use category
+       \required-field
+       \type choice
+       \key Heating
+       \key Cooling
+       \key InteriorLights
+       \key ExteriorLights
+       \key InteriorEquipment
+       \key ExteriorEquipment
+       \key Fans
+       \key Pumps
+       \key HeatRejection
+       \key Humidifier
+       \key HeatRecovery
+       \key WaterSystems
+       \key Refrigeration
+       \key OnSiteGeneration
+       \key HeatingCoils
+       \key CoolingCoils
+       \key Chillers
+       \key Boilers
+       \key Baseboard
+       \key HeatRecoveryForCooling
+       \key HeatRecoveryForHeating
+  A9, \field End-Use Subcategory
+       \type alpha
+       \note enter a user-defined subcategory for this metered output
+  A10; \field Units
+       \note optional but will result in dimensionless units for blank
+       \note EnergyPlus units are standard SI units
+       \type alpha
+
+OS:EnergyManagementSystem:TrendVariable,
+       \memo This object sets up an EMS trend variable from an Erl variable
+       \memo A trend variable logs values across timesteps
+       \min-fields 4
+  A1, \field Handle
+       \type handle
+       \required-field      
+  A2, \field Name
+       \required-field
+       \type alpha
+       \note No spaces allowed in names for ErlVariableNames
+       \reference ErlVariableNames
+  A3, \field EMS Variable Name
+       \required-field
+       \type object-list
+	   \object-list ErlVariableNames
```

Comment:
got it.  yep, these are still ErlVariableNames since they must be GlobalVariables